### PR TITLE
Fix warning in ThreadSafeDynamicsWorld

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: libraries/physics/src/PhysicsEngine.cpp [hunk 1]
```diff
@@ -228,7 +228,7 @@ void PhysicsEngine::removeContacts(ObjectMotionState* motionState) {
 
 void PhysicsEngine::stepSimulation() {
     CProfileManager::Reset();
-    BT_PROFILE("stepSimulation");
+    BT_PROFILE("stepSimulationWithSubstepCallback");
```
Comment (ZappoMan) [inline libraries/physics/src/PhysicsEngine.cpp]: did you want to do this? seems like maybe a misplaced global search/replace

File: libraries/physics/src/PhysicsEngine.cpp [hunk 2]
```diff
@@ -241,7 +241,7 @@ void PhysicsEngine::stepSimulation() {
     float timeStep = btMin(dt, MAX_TIMESTEP);
 
     if (_myAvatarController) {
-        // ADEBUG TODO: move this stuff outside and in front of stepSimulation, because 
+        // ADEBUG TODO: move this stuff outside and in front of stepSimulationWithSubstepCallback, because
```
Comment (ZappoMan) [inline libraries/physics/src/PhysicsEngine.cpp]: maybe this too should not be renamed

File: libraries/physics/src/PhysicsEngine.cpp (hunk 1 above)
Comment (huffman) [inline libraries/physics/src/PhysicsEngine.cpp]: Indeed, I thought this was in the renamed method - fixed!

**Comment by ZappoMan** [comment]
will merge once first build finishes

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7a4787a79cec2d33471ae86ebbb27f511705e2d8 available at: https://gist.github.com/207f286de029751ac4d1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6241.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6241.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 84869c512b44ac1ca27b2e6cadd2616ac0ea865c available at: https://gist.github.com/207f286de029751ac4d1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6241.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6241.dmg

**Pull request merged**